applications/app/views/fragments/mediaBody.scala.html

```diff
@@ -76,6 +85,10 @@
                                                 <meta itemprop="thumbnail" content="@SeoOptimisedContentImage.bestFor(img.images)" />
                                                 <meta itemprop="thumbnailUrl" content="@SeoOptimisedContentImage.bestFor(img.images)" />
                                             }
+                                            @video.mediaAtom.map { mediaAtom =>
+                                              @fragments.atoms.media(mediaAtom)
+                                            }
+
```

Comment:
If a video page were to include both a Video Element and a Media Atom both would currently be displayed. Though this shouldn't happen if people are concerned about this I can use an if statement to only display one or the other.